.github/workflows/sycl_windows_build_and_test.yml

```diff
@@ -2,54 +2,80 @@ name: SYCL Windows Test
 
 on:
   workflow_call:
+    inputs:
+      build_cache_suffix:
+        type: string
+        required: false
+        default: "default"
 
 jobs:
   build:
     name: Build
-    runs-on: windows-2022
+    runs-on: [Windows, build]
+    # TODO use cached checkout
     steps:
-    - uses: actions/checkout@v2
-      with:
-        path: src
-        fetch-depth: 1
-    - name: Install dependencies
-      shell: cmd
-      run: |
-        choco install -y ninja
-        choco install -y sccache --version 0.2.15
-        refreshenv
     - uses: ilammy/msvc-dev-cmd@9f8ae839b01883414208f29e3e24524387f48e1f
```

Comment:
Why SHA instead of a release version?

Reply:
Tags can change, commit SHA will always point to the same version of code.